### PR TITLE
feat: add tips of the day for new sessions

### PR DESCRIPTION
**Pull request by michaelneale**
This adds some tips to the CLI when it first shows up for a new session. 

As @alecthomas says: "nobody is reading a readme anymore"

### CODE REVIEW

File: src/goose/cli/session.py [hunk 1]
```diff
@@ -27,6 +28,30 @@
 RESUME_MESSAGE = "I see we were interrupted. How can I help you?"
 
 
+TIPS = [
```
Comment (alecthomas) [inline src/goose/cli/session.py:31]: Did Goose write these tips? 🤔

**Review by alecthomas — APPROVED** [review]

File: src/goose/cli/session.py (hunk 1 above)
Comment (michaelneale) [reply src/goose/cli/session.py:31]: no... in this case genuine human

**Comment by alecthomas** [comment]
I've got a solution for your conventional commits woes! https://github.com/alecthomas/gptcc

And an [example](https://github.com/TBD54566975/ftl/blob/main/.github/workflows/cc.yml) of using it.

**Comment by michaelneale** [comment]
closing this - as I think we can have a better interactive first time tutorial approach

**Pull request closed without merging**